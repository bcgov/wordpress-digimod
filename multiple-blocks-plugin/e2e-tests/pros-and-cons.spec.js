import {
    createNewPost,
    enablePageDialogAccept,
    insertBlock,
    getEditedPostContent
} from '@wordpress/e2e-test-utils';

describe( 'Pros and cons block', () => {
    jest.setTimeout(60000);

    beforeAll( async () => {
        await enablePageDialogAccept();
    } );
    beforeEach( async () => {
        await createNewPost();
    } );

    it( 'Pros and cons block should be available', async () => {
        await insertBlock( 'pros-and-cons' );
		// Check if block was inserted
        
		expect( await page.$( '[data-type="multiple-blocks-plugin/pros-and-cons"]' ) ).not.toBeNull();

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );