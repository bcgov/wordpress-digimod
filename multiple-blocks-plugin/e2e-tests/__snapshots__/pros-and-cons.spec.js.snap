// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pros and cons block Pros and cons block should be available 1`] = `
"<!-- wp:multiple-blocks-plugin/pros-and-cons -->
<div class=\\"row\\"><!-- wp:multiple-blocks-plugin/dm-pros-list -->
<div class=\\"col-sm-12 col-md-6\\"><!-- wp:list {\\"className\\":\\"prosList\\"} -->
<ul class=\\"prosList\\"><!-- wp:list-item -->
<li></li>
<!-- /wp:list-item --></ul>
<!-- /wp:list --></div>
<!-- /wp:multiple-blocks-plugin/dm-pros-list -->

<!-- wp:multiple-blocks-plugin/dm-cons-list -->
<div class=\\"col-sm-12 col-md-6\\"><!-- wp:list {\\"className\\":\\"consList\\"} -->
<ul class=\\"consList\\"><!-- wp:list-item -->
<li></li>
<!-- /wp:list-item --></ul>
<!-- /wp:list --></div>
<!-- /wp:multiple-blocks-plugin/dm-cons-list --></div>
<!-- /wp:multiple-blocks-plugin/pros-and-cons -->"
`;
